appinventor/appengine/src/com/google/appinventor/client/editor/youngandroid/BlocklyPanel.java

```diff
@@ -463,6 +455,32 @@ public static void callToggleWarning() {
     YaBlocksEditor.toggleWarning();
   }
 
+  /**For Walkthroughs**/
```

Comment:
would be nice to have a short explanation of what the code does, or how it flows (when it starts, what's next, and so forth).